src/Footer/Footer.js

```diff
@@ -8,8 +9,28 @@ import {
 } from '@folio/stripes/components';
 
 import css from './Footer.css';
+import { ALLOWED, DENIED } from '../../constants';
```

Comment:
Lets follow general import order.
Would be great to import css file at the end of all imports.